day24/pytest/cornelius/README.adoc

```diff
@@ -0,0 +1,63 @@
+== Day 24: Lobby Layout
+
+This solution is written as test using pytest.
+
+The original puzzle can be found at https://adventofcode.com/2020/day/24.
+
+For execution enter `pytest` on the command line (your working directory has to be this directory where the test is). It will pick up all files where the filename starts with `test_`.
+
+=== Conway's World in Hex
+
+The lobby looks a bit like the world we did on https://github.com/docToolchain/aoc-2020/tree/main/day17/pytest/cornelius[day 17]. It's two-dimensional and the elements are hexagonal, but otherwise it can be represented in the same way with a slight reinterpretation of the coordinate system.
+
+image::coordinate-system.png[Coordinate System]
```

Comment:
This reference doesn't seem to work. The image doesn't show up in the preview. How can I fix that? Is this a matter of adapting the path?